README.md

```diff
@@ -1,3 +1,7 @@
+> Codacy Analysis CLI is deprecated
+>
+> Use [Codacy CLI V2](https://github.com/codacy/codacy-cli-v2) instead
```

Comment:
Maybe we need to specify something like:

- For uploading results to Codacy fir client side rules use this CLI v1 - [Client Side Tools](https://docs.codacy.com/repositories-configure/local-analysis/client-side-tools/)
- For IDEs or AI usage, please rely on the new [Codacy CLI V2](https://github.com/codacy/codacy-cli-v2) instead